react/Footer/Footer.common.less

```diff
@@ -0,0 +1,25 @@
+@import (reference) "~seek-style-guide/theme";
+
+@footer-vertical: ~"and (max-width: 767px)";
```

Comment:
Yep. Phase one is to extract the current header and footer, cleaning up the code to an acceptable level, but otherwise leaving it alone.